docs/embed.md

```diff
@@ -0,0 +1,196 @@
+is true for any other use of the URL, e.g. bookmarking or social sharing.
+
+This is not necessarily an issue. You can just live with it. It is also
+possible to write some JavaScript to encode some additional state in the outer
+URL. But it is very hard to do that without knowing something about the
+embedding website.
+
+### Phishing
+
+Phishing is generally countered by checking the URL in the browser. The URL of
+iframes is not visible by default, though. This means that any security related
+actions (e.g. login) should not be done in the iframe, but in a pop-up or on
+the platform.
+
+### Social Share
+
+When a user shares something on facebook or other social media, the information
+that is actually shared is a URL. The social media site will also try to
+extract some additional metadata (title, description, image) from that URL.
+
+If we think about embedding, it is important to ask: Which URL do we want to
+share (platform or embedded) and how can we provide the metadata? The answer to
+the first question is: It depends.
+
+If we share the platform URL, we can just add the metadata to that page. But if
+the shared URL points to the page where our content is embedded, we have no
+control over the metadata. If we want to have that control, we can provide an
+additional service that just provides metadata and redirects for social media
+and redirects humans to the embed.
+
+### Third Party Cookies
+
+Websites can set cookies in the browser to preserve some user-specific state,
+most importantly their session. Without them, it would be impossible to log in.
+
+There is a distinction between 1st-party-cookies (set by the page you are
+currently visiting) and 3rd-party-cookies (set by other pages).
+3rd-party-cookies are often used by advertisement companies to track users. So
+it is a common privacy recommendation to disable them.
+
+When embedded, we also are a 3rd party, so our cookies may get blocked. There
+is no practical way around this other than requiring users to enable
+3rd-party-cookies.
+
+### Broken Visual Hierarchy
+
+If the embed has its own header with a logo and login, it might look out of
+place when embedded. You need to carefully craft the embed UI to make it work
+in different contexts.
+
+### Public API
+
+Once people start embedding your page, the embedding API needs to be stable. In
+most cases it is close to impossible to notify everyone to change their embed
+codes.
+
+### Account Integration
+
+The whole point of embedding is lowering the entry barrier for new users. This
+works even better if the account system is integrated with the embedding
+website so that no additional login is required for participation.
+
+## Possible extensions
+
+-   Currently, it is only possible to embed full-functioning projects. Other
+    options may be added under the `/embed/` prefix, optionally using the embed
+    shell.
+-   The embed shell sends an additional `X-Embed` header with every request.
```

Comment:
The header is implemented. There are simply no views yet that react to it. Also note that we are still working on this story and more features will be implemented during this sprint.